content/updates/2019-09-06-depreecation-policy.md

```diff
@@ -0,0 +1,38 @@
++++
+date = "2019-08-26"
+title = "For Humans and Agencies: The cloud.gov Deprecation Policy" 
++++
+
+# For Humans and Agencies: The cloud.gov Deprecation Policy
+
+We'd like to talk about two aspects of being a hosting provider: consistency and predictability. In the context of being a hosting provider, consistentcy is the ability to do something the same way every time, and predictability means you can count on something to happen in a very specific way. We have been neither consistent nor predictable, and we'd like to talk about how things are changing and heading towards a positive direction.
+
+Firstly, this is the formal announcement of our deprecation policy, found [here](). Our deprecation policy is dual-focused decision; a business decision on how we want to be consistent, and human decision on how we want our customers to trust us. We recommend you take some time to read up on the specific of the policy, and how it can impact your business.
+
+## Consistency
+
+As a hosting provider, consistency is very important, because that provides predictability. Thus far, over the years, we've had to deprecate several aspects of the system as technologies grow, change, or our dependencies get deprecated. Looking back, in every single instance of needing to deprecate something, the cloud.gov team has had to sit down and have a meeting on how we want to handle a specific deprecation. We discuss things like the impact to ourselves, customers, and technologies.
+
+We came to realise our deprecation policy had been, _what do we do this time?_, instead of following a policy which had been set. This provided an inconsistent focus on how we wanted to present news, information, and execution of the given deprecation. The presented both problems for us and customers, as we weren't consistent with messaging or execution, and customers were often not supported as well as they could have been through consistent messaging and consistent support.
```

Comment:
Last sentence should probably stop at "messaging or execution" - the second clause is mostly a restatement of the first and using the word "consistent" three times in one sentence feels heavy-handed while reading it